### PR TITLE
profiles: Update Edinburgh Symbology

### LINKED ISSUE #293: Update EGPH Profile
Update profile to new symbology
Real world display:
image
image

I've completed this if someone could assign, mine looks like this:
image

Similar to Manchester I could update the colour of the FAVA to reflect the change however the land colour would have to be changed manually from the default to look exactly the same.

### PR DESCRIPTION
**Pull request by SamLefevre**
Fixes #293 

# Summary of changes

Update Symbology to new style

# Screenshots (if necessary)

Radar 1
image

Radar 2
image

Radar 3
image

Radar 4
image

### CODE REVIEW

**Comment by luke11brown** [comment]
Given the Sc_App General file has been modified, how does this affect the Glasgow displays?

File: UK/Data/Settings/Local/Edinburgh/Symbology.txt [hunk 1]
```diff
@@ -97,20 +97,20 @@ Ground Network:taxiway:65280:3.5:0:3:7
 Ground Network:terminal taxiway:255:3.5:0:3:7
 Sector:line:21331:0.5:2:2:7
 Sector:msaw:32768:2.0:0:2:7
-Sector:active sector background:5519132:3.5:0:0:7
-Sector:inactive sector background:5519132:3.5:0:0:7
+Sector:active sector background:8747638:3.5:0:0:7
```
Comment (luke11brown) [inline UK/Data/Settings/Local/Edinburgh/Symbology.txt:100]: I'm not sure these colours are reflective of the InNOVA displays... 
I'd be careful of just colour picking from what is possibly an over/under exposed image of the real system.
I'd like some input from the users of the real system to compare.

File: UK/Data/Settings/Sc_APP/Sc_APP_General.txt [hunk 2]
```diff
@@ -125,13 +125,6 @@ m_RubberHeadingRounding:5
 SqawkForSModeTransponder:
 m_TransparentTagBackground:1
 m_UseArrDepTagBgColor:0
-SET_MTCA_Show:1
-m_TaxiOutTime:7
-m_StartupTime:10
-m_ConnectTime:15
-m_AutoAssumeOnGroundStateChangeButton:0
```
Comment (luke11brown) [inline UK/Data/Settings/Sc_APP/Sc_APP_General.txt:132]: These settings, related to ES r29 need to remain

**Review by luke11brown — CHANGES_REQUESTED** [review]
Please see above comments

**Comment by SamLefevre** [comment]
> Given the Sc_App General file has been modified, how does this affect the Glasgow displays?

The Glasgow profiles at the moment use the GEO extended centrelines, so the profile shouldn't be effected

**Review by luke11brown — APPROVED** [review]

**Comment by luke11brown** [comment]
Still unsure about colour selection

**Comment by SamLefevre** [comment]
image
image

Made the inactive and active colour more like real world, however the default land colour doesn't make it look great

**Pull request merged**